packages/node/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import { AddressZero } from "ethers/constants";
```

Comment:
We might not want this but having a place to start gathering all the constants across the codebase might be a reasonable idea.